restapi/utils.go

```diff
@@ -106,12 +107,14 @@ func FileExists(filename string) bool {
 }
 
 func NewSessionCookieForConsole(token string) http.Cookie {
+	sessionDuration := GetConsoleCookieDurationInSeconds()
+	expiresAt := time.Duration(int64(time.Second) * int64(ranx.Int31n(int32(sessionDuration))))
 	return http.Cookie{
 		Path:     "/",
 		Name:     "token",
 		Value:    token,
-		MaxAge:   int(SessionDuration.Seconds()), // 45 minutes
-		Expires:  time.Now().Add(SessionDuration),
+		MaxAge:   sessionDuration, // default 1 hr
+		Expires:  time.Now().Add(expiresAt * time.Second),
```

Comment:
```suggestion
		MaxAge: int(sessionDuration.Seconds()),
		Expires:  time.Now().Add(sessionDuration),
```

Also we can only set one we don't need to set both

```
   If a cookie has both the Max-Age and the Expires attribute, the Max-
   Age attribute has precedence and controls the expiration date of the
   cookie.  If a cookie has neither the Max-Age nor the Expires
   attribute, the user agent will retain the cookie until "the current
   session is over" (as defined by the user agent).
```

https://datatracker.ietf.org/doc/html/rfc6265